src/main/java/com/gmail/inverseconduit/commands/sets/AdminCommands.java

```diff
@@ -0,0 +1,152 @@
+package com.gmail.inverseconduit.commands.sets;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.gmail.inverseconduit.BotConfig;
+import com.gmail.inverseconduit.commands.CommandHandle;
+
+public class AdminCommands {
+
+	public static CommandHandle addAdminCommand(BotConfig bc) {
+		return new CommandHandle.Builder("addAdmin", message -> {
+
+			if (isElevatedUser(message.getUserId(), bc)) {
+				String[] args = message.getMessage().trim().split(" ");
+				try {
+					Integer newUserId = Integer.parseInt(args[1]);
+					if (!bc.getAdmins().contains(newUserId))
+						bc.getAdmins().add(newUserId);
+					else return "User already has elevated privileges!";
+					Path file = Paths.get("bot.properties");
+					addAdminToFile(file, newUserId);
+					return "Added userID " + newUserId + " successful!";
+				} catch (NumberFormatException e) {
+					return "Could not understand userID!";
+				}
+			}
+			return "I am afraid, I cannot let you do that";
+
+		}).build();
+	}
+
+	public static CommandHandle removeAdminCommand(BotConfig bc) {
+		return new CommandHandle.Builder("removeAdmin", message -> {
+
+			if (isElevatedUser(message.getUserId(), bc)) {

```

Comment:
same goes for here... would it be viable to extract the privilege-check into the CommandHandle itself?